index.js

```diff
@@ -538,10 +546,19 @@ function parse (args, opts) {
     argv: argv,
     error: error,
     aliases: flags.aliases,
-    newAliases: newAliases
+    newAliases: newAliases,
+    configuration: configuration
   }
 }
 
+function loadConfiguration () {
+  var pkg = readPkgUp.sync({cwd: path.dirname(parentModule)}).pkg
+  return assign({
```

Comment:
I was thinking about adding a `defaults` option to https://github.com/sindresorhus/pkg-conf. Would that make you able to use it?